ocaml/typing/typemod.ml

```diff
@@ -3221,38 +3348,40 @@ and type_structure ?(toplevel = None) funct_body anchor env sstr =
                 ])
              classes []),
         shape_map,
-        new_env
+        new_env,
+        subst
     | Pstr_include sincl ->
-        type_str_include ~functor_:false ~loc env shape_map sincl sig_acc
+        let (tstr, sg, map, newenv) = type_str_include ~functor_:false ~loc env shape_map sincl sig_acc in
+        tstr, sg, map, newenv, subst
```

Comment:
I think we will support `include` for substitution - if it's too much work, let's add a CR.